### PR TITLE
Make smaller and more precise queries for Prow usage

### PR DESCRIPTION
**Pull request by stevekuznetsov**
The query output is identical and there is no change to the dashboard.

Signed-off-by: Steve Kuznetsov <skuznets@redhat.com>

/assign @hongkailiu @droslean

### CODE REVIEW

File: cluster/ci/monitoring/mixins/grafana_dashboards/prow.jsonnet [hunk 1]
```diff
@@ -165,104 +69,14 @@ dashboard.new(
    ) + legendConfig)
   .addTarget(
     prometheus.target(
-      'sum(container_memory_working_set_bytes{namespace="ci",container_name!="POD"} * on (pod_name) group_left(pod) label_replace(kube_pod_labels{pod!="",label_app="prow", label_component="hook"}, "pod_name", "$1", "pod", "(.*)"))',
+      '',
```
Comment (hongkailiu) [inline cluster/ci/monitoring/mixins/grafana_dashboards/prow.jsonnet]: Leftover?

Comment (stevekuznetsov) [reply cluster/ci/monitoring/mixins/grafana_dashboards/prow.jsonnet]: lol

**Pull request merged**